README.md

```diff
@@ -63,7 +53,7 @@ NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=<your_wallet_connect_project_id_here>
 ``` 
```

Comment:
I think this backlashes should go in line 41 after `yarn install`. 
This is causing line 42 `### Managing... ` look like that is part of the commands.

Reply:
it's already after yarn install

Reply:
Those are env variables that you add ..